tka/chaintest_test.go

```diff
@@ -30,13 +30,21 @@ type testchainNode struct {
 	HashSeed   int
 	Template   string
 	SignedWith string
+
+	// When set, uses this hash as the parent hash when
+	// Parent is not set.
+	//
+	// Set when a testChain is based on a different one
+	// (in scenario_test.go).
+	ParentHash AUMHash
```

Comment:
"When set" does this mean an AUMHash of zero bytes is unset?

I'm fine with that, but elsewhere you have used pointers-to-thing and nil to represent unset. Probably worth being consistent even in tests.

Reply:
Done